packages/atomic/src/components/insight/atomic-insight-interface/e2e/atomic-insight-interface.e2e.ts

```diff
@@ -41,15 +41,18 @@ test.describe('Atomic Insight Panel', () => {
   test.describe('result actions interactions', () => {
     test.beforeEach(async ({insightInterface}) => {
       // Video results have better actions to test
+      const insightSearchPromise =
+        insightInterface.waitForInsightSearchResponse();
       await insightInterface.getTabByName('Videos').click();
-      await insightInterface.waitForNonVideoResultsToBeDetached();
+      await insightSearchPromise;
+      await insightInterface.waitForVideoResultLinksToBeVisible();
```

Comment:
@alexprudhomme Only way I've found to be sure that the results would actually be displayed after changing Tab.

We don't want the Videos tab to be first, that's not the point of this test.

So we wait until we've got the response from the insight/search, and then make sure Video results are rendered.

And then we can hover over results.